src/python/pants/option/global_options.py

```diff
@@ -183,6 +183,8 @@ def register_bootstrap_options(cls, register):
     register('--subproject-roots', type=list, advanced=True, fromfile=True, default=[],
              help='Paths that correspond with build roots for any subproject that this '
                   'project depends on.')
+    register('--owner-of', help='Select the targets that own these files', type=list, default=[],
```

Comment:
This should mention the existence of the `--changed` options, and the fact that the three types are exclusive.